### PR TITLE
ReadTheDocs

### PR DESCRIPTION
**Pull request by carlosggarcia**
Creating documentation with readthedocs

### CODE REVIEW

File: doc/source/conf.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = "TJPCov"
+copyright = "2019, LSST DESC"
```
Comment (mattkwiecien) [inline doc/source/conf.py:21]: Should this be 2022 (or 2023)?

File: tests/test_covariance_calculator.py [hunk 2]
```diff
@@ -83,7 +83,11 @@ def test_create_sacc_cov():
     cc = CovarianceCalculator(input_yml)
```
Comment (mattkwiecien) [inline tests/test_covariance_calculator.py:83]: You could use a fixture here and use dependency injection to pass the cov calc into each test method.  

```
@pytest.fixture
def mock_covariance():
  return CovarianceCalculator(input_yml)

def my_test(mock_covariance):
   mock_covariance.get_covariance()
```
etc.  Definitely not required though, but something I came across when reading up on pytest.

File: tests/test_covariance_calculator.py [hunk 3]
```diff
@@ -83,7 +83,11 @@ def test_create_sacc_cov():
...
     cov = cc.get_covariance() + 1e-100
 
-    cc.create_sacc_cov()
+    # Check returned file
+    s = cc.create_sacc_cov()
+    cov2 = s.covariance.covmat + 1e-100
+    assert np.max(np.abs(cov / cov2 - 1) < 1e-10)
```
Comment (mattkwiecien) [inline tests/test_covariance_calculator.py:89]: There's also a pytest way of doing these almost equals:

```
assert test == pytest.approx(ref, 1e-10)
```
Definitely not required change though

File: tjpcov/covariance_calculator.py [hunk 4]
```diff
@@ -147,29 +143,30 @@ def get_covariance_terms(self):
...
+        """Write the sacc file with the total covariance.
+
+        Args:
+            output (str, optional): Filename. This will be joined to the outdir
+                path specified in the configuration file. Default
+                "cls_cov.fits"
+            save_terms (bool, optional): If true, save individual files for
+                each of the requested contributions. The will have the
+                covariance term (e.g. gauss) appended to the filename (before
+                the extension, e.g. cls_cov_gauss.fits)
```
Comment (mattkwiecien) [inline tjpcov/covariance_calculator.py:155]: Maybe add a Returns: part to the docstring now that we are returning?

File: tjpcov/wigner_transform.py [hunk 5]
```diff
@@ -432,22 +400,18 @@ def wigner_d_parallel(s1, s2, theta, ell, ncpu=None, l_use_bessel=1.0e4):
...
+    This function works on both one dimensional vectors and two dimensional
+    covariance covrices.
+
+    Args:
+        r: theta or ell values at which the un-binned vector is computed.
+        cov: Unbinned covariance. It also works for a vector of C_ell or xi
+        r_bins: theta or ell bins to which the values should be binned.
+
+    Returns:
+        array: Binned covariance or vector of C_ell or xi
```
Comment (mattkwiecien) [inline tjpcov/wigner_transform.py:414]: Also very minor but I think array should be array_like

**Review by mattkwiecien — APPROVED** [review]
Looks good, left some minor optional comments

File: doc/source/conf.py (hunk 1 above)
Comment (carlosggarcia) [reply doc/source/conf.py:21]: No idea of what should go there. Maybe 2019-2023?

File: tests/test_covariance_calculator.py (hunk 2 above)
Comment (carlosggarcia) [reply tests/test_covariance_calculator.py:83]: Interesting. I'll open an issue to improve the unit test with that but for now I'd rather touch as little as possible the actual code.

File: tests/test_covariance_calculator.py (hunk 3 above)
Comment (carlosggarcia) [reply tests/test_covariance_calculator.py:89]: I'll leave it like that for now but looks pretty convenient. Thanks!

File: tjpcov/covariance_calculator.py (hunk 4 above)
Comment (carlosggarcia) [reply tjpcov/covariance_calculator.py:155]: Done

File: tjpcov/wigner_transform.py (hunk 5 above)
Comment (carlosggarcia) [reply tjpcov/wigner_transform.py]: Done.

File: doc/source/conf.py (hunk 1 above)
Comment (carlosggarcia) [reply doc/source/conf.py]: I've added a (dynamic) range based on this: https://www.copyrightlaws.com/copyright-symbol-notice-year/

**Pull request merged**